File: ActivityTutorial/app/src/main/java/com/issam/drmas/activitytutorial/MainActivity.java
package com.issam.drmas.activitytutorial;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void PassVariables(View view){
        startActivity(new Intent(MainActivity.this, PassVariablesActivity.class));
    }

    public void startActivityForResult(View view){
        startActivity(new Intent(MainActivity.this, StartActivityForResultActivity.class));
    }
}

File: MyKotlin2/app/src/main/java/com/issam/drmas/mykotlin/MainActivity.kt
package com.issam.drmas.mykotlin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.issam.drmas.mykotlin.Calculator.CalculatorActivity
import com.issam.drmas.mykotlin.FindAge.FindAgeActivity
import com.issam.drmas.mykotlin.FoodApp.FoodActivity
import com.issam.drmas.mykotlin.TicTac.TicTacActivity
import java.time.Instant

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun findAge(view:View){
       val intent = Intent(this, FindAgeActivity::class.java)
        startActivity(intent)
    }
    fun ticTac(view: View){
        val intent = Intent(this, TicTacActivity::class.java)
        startActivity(intent)
    }
    fun CalculatorClicked(view: View){
        val intent = Intent(this, CalculatorActivity::class.java)
        startActivity(intent)
    }
    fun FoodAppClicked(view: View){
        val intent = Intent(this, FoodActivity::class.java)
        startActivity(intent)
    }
}
